lib/sms.rb

```diff
@@ -0,0 +1,22 @@
+require "twilio-ruby"
+require_relative "authentication"
```

Comment:
I think this was checked in by mistake, I had to remove it to get the tests to run (good test coverage though).

Reply:
Yes, this was my own file...I am not sure how to tell someone that they need to create their own file and require it here for the sms sending to work....what approach would you suggest?